ThrowIT/ViewControllers/TimelineViewController.m

```diff
@@ -181,7 +183,31 @@ - (NSInteger)tableView:(nonnull UITableView *)tableView numberOfRowsInSection:(N
 }
 
 -(void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath *)indexPath{
-    [tableView deselectRowAtIndexPath:indexPath animated:YES];
+    self.tableSelection = indexPath;
```

Comment:
instead of storing this as an ivar (instance var). could we pass it as a param to the respective methods?